src/casp/services/model_inference/server.py

```diff
@@ -0,0 +1,35 @@
+import multiprocessing
+import typing as t
+
+import pandas as pd
+import uvicorn
+from fastapi import FastAPI, File, Form
+
+from casp.services.db import ops
+from casp.services.model_inference import utils
+
+app = FastAPI()
+
+
+@app.post("/predict")
+async def predict(file: bytes = File(), model_system_name: str = Form()) -> t.Dict:
+    # Get Model dump file
+    model_info = ops.get_model_by(system_name=model_system_name)
+    dump_manager = utils.get_dump_manager(model_info.model_file_path)
+    model = dump_manager.model
+    transform = dump_manager.transform
```

Comment:
The plan is to bundle model and transform into one: even though transform would be a standalone `nn.Module`, it will be called automatically with the model's `.predict` method.

Can you make an issue to the tune of "better integration of `cellarium-ml` and `cellarium-cloud`"? In that feature, we should also refactor/remove model implementations out of `cellarium-cloud`.